lib/android/ui/ui.nit

```diff
@@ -17,134 +15,123 @@
 # Views and services to use the Android native user interface
 module ui
 
+# Implementation note:
+#
+# We cannot rely on `Activity::on_restore_instance_state` to implement
+# `on_restore_state` is it only invoked if there is a bundled state,
+# and we don't use the Android bundled state.
```

Comment:
It says that we do not rely on the corresponding Android annotation because we do not fully use their save/restore system, so it is unreliable. It's more of a note to a future me, or another Android developer wondering why on_restore_instance_state is not used.